timbuctoo-instancev4/src/main/java/nl/knaw/huygens/timbuctoo/rdf/tripleprocessor/CollectionMembershipTripleProcessor.java

```diff
@@ -16,6 +16,7 @@ public void process(String vreName, boolean isAssertion, Triple triple) {
 
     if (isAssertion) {
       entity.addToCollection(database.findOrCreateCollection(vreName, triple.getObject()));
+      entity.removeFromCollection(database.getDefaultCollection(vreName));
```

Comment:
I do think that means that an entity that is added to a collection will always have only the base-collection at that point. Simplifying your last commit. Am I right?